### DISCUSSION TITLE
Play a live audio (stream)

### DISCUSSION THREAD

**Discussion by matteomontipo** [lhz516/react-h5-audio-player#163] [Q&A] [answered]
Is there a way to play live audio urls with this? sort of what you achieve by using `<source>` tag within a standard html5 audio player

**Comment by lhz516** [comment, marked as answer]
It's not supported currently.

**Reply by johnrisby** [reply]
Hi, is this now supported? I just tested it with a live steam (an internet radio station via an m3u file) and it plays fine.

Is there a way to get meta data such as current track played? Thanks
